src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -391,7 +400,9 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 	}
 
 	public onEnterSet() {
-		this.codeActionList.value?.setSelection([this.currSelectedItem]);
+		if (this.allowEnterClick) {
```

Comment:
Having `allowEnterClick` and `currSelectedItem` seems  redundant. Could `currSelectedItem` be nullable instead to indicate nothing is selected and that nothing should be accepted on enter?